### ISSUE TITLE
`ObservationLong` accepts a parameter to automatically overwrite old visiblities

### ISSUE THREAD

**Issue by kenfus** [i4Ds/Karabo-Pipeline#382]
Running a `ObservationLong` on cscs sometimes fails with the following error:

```
Some example files to remove/replace:
['/scratch/snx3000/vtimmel/data/visibilities.ms/beam_vis_24.vis', '/scratch/snx3000/vtimmel/data/visibilities.ms/beam_vis_4.vis', '/scratch/snx3000/vtimmel/data/visibilities.ms/beam_vis_8.vis', '/scratch/snx3000/vtimmel/data/visibilities.ms/beam_vis_18.ms', '/scratch/snx3000/vtimmel/data/visibilities.ms/beam_vis_19.ms', '/scratch/snx3000/vtimmel/data/visibilities.ms/beam_vis_23.ms']
Traceback (most recent call last):
  File "/users/vtimmel/karabo/Karabo-Pipeline/time_karabo_slurm.py", line 146, in <module>
    main(
  File "/users/vtimmel/karabo/Karabo-Pipeline/time_karabo_slurm.py", line 128, in main
    visibilities = interferometer_sim.run_simulation(telescope, sky, observation_long)
  File "/users/vtimmel/karabo/Karabo-Pipeline/karabo/simulation/interferometer.py", line 202, in run_simulation
    return self.__run_simulation_long(
  File "/users/vtimmel/karabo/Karabo-Pipeline/karabo/simulation/interferometer.py", line 340, in __run_simulation_long
    raise exp
  File "/users/vtimmel/karabo/Karabo-Pipeline/karabo/simulation/interferometer.py", line 280, in __run_simulation_long
    ans = input_wrapper(msg=msg, ret="y")
  File "/users/vtimmel/karabo/Karabo-Pipeline/karabo/util/data_util.py", line 118, in input_wrapper
    return input(msg)
EOFError: EOF when reading a line
Found already existing "beam_vis_*.vis" and beam_vis_*.ms" files inside /scratch/snx3000/vtimmel/data/visibilities.ms,                         + Do you want to replace remove/replace them? [y/N]srun: error: nid01945: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=45602166.0
```

I have never seen this error locally, and sometimes it also works on CSCS. Maybe the popup-window asking you what to do sometimes causes a problem on cscs? I think this should be changed to a parameter anyway, or save the visibilities in a datetime folder instead of overwriting old runs.

**Comment by deiruch** [comment]
I think `uuid > datetime`, and we should use that as default value for the parameter. Also, perhaps `/tmp` might be a better default location?

**Comment by kenfus** [comment]
Closed with PR #386
